lib/buffer.js

```diff
@@ -14,11 +15,19 @@ Buffer.poolSize = 8 * 1024;
 var poolSize, poolOffset, allocPool;
 
 
+binding.setupBufferJS(Buffer.prototype, bindingObj);
+const flags = bindingObj.flags;
```

Comment:
What do you think about making this one-byte Uint8Array instead, and passing it to the C++?